src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs

```diff
@@ -103,7 +103,14 @@ public override void ExecuteCmdlet()
                     azureWorkloadRecoveryConfig.RecoveryPoint = RecoveryPoint;
                     Dictionary<UriEnums, string> keyValueDict = HelperUtils.ParseUri(RecoveryPoint.Id);
                     string containerUri = HelperUtils.GetContainerUri(keyValueDict, RecoveryPoint.Id);
-                    targetServer = containerUri.Split(new string[] { ";" }, StringSplitOptions.None)[3];
+                    try
+                    {
```

Comment:
Similarly here. We have two types of containers. To accommodate SqlAgContainers too, this change was made.